packages/kbn-scout/src/playwright/fixtures/test/page.ts

```diff
@@ -28,10 +28,14 @@ function extendPageWithTestSubject(page: Page) {
     'innerText',
     'isChecked',
     'isHidden',
+    'isVisible',
     'locator',
+    'waitForSelector',
   ];
 
-  const extendedMethods: Partial<Record<keyof Page, Function>> = {};
+  const extendedMethods: Partial<Record<keyof Page, Function>> & {
```

Comment:
I dont get this type man.  
So, it's a partial record of all function elements within the Page type?

Reply:
I get what it accomplishes, but not the impl.